doc/computing/parallelism.rst

```diff
@@ -179,83 +181,91 @@ Environment variables
 
 These environment variables should be set before importing scikit-learn.
 
-:SKLEARN_ASSUME_FINITE:
+`SKLEARN_ASSUME_FINITE`
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Sets the default value for the `assume_finite` argument of
+:func:`sklearn.set_config`.
+
+`SKLEARN_WORKING_MEMORY`
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sets the default value for the `working_memory` argument of
+:func:`sklearn.set_config`.
+
+`SKLEARN_SEED`
+~~~~~~~~~~~~~~
+
+Sets the seed of the global random generator when running the tests, for
+reproducibility.
+
+Note that scikit-learn tests are expected to run deterministically with
+explicit seeding of their own independent RNG instances instead of relying on
+the numpy or Python standard library RNG singletons to make sure that test
+results are independent of the test execution order. However some tests might
+forget to use explicit seeding and this variable is a way to control the intial
+state of the aforementioned singletons.
+
+`SKLEARN_TESTS_GLOBAL_RANDOM_SEED`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
For the links to work:

```suggestion
.. _sklearn_tests_global_random_seed:

`SKLEARN_TESTS_GLOBAL_RANDOM_SEED`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
This already works for non-restructured text links that I use in the test report output / exception message:

https://181804-843222-gh.circle-artifacts.com/0/doc/computing/parallelism.html#sklearn-tests-global-random-seed

Reply:
We could add the explicit ref for restructed text reference linking but I don't think we really need it in this case.

As you prefer.

Reply:
Ah I see. I didn't know that sphinx auto-converts the section header into a link. I'm okay with leaving the ref out.